src/routes/plans/+page.svelte

```diff
@@ -970,6 +970,7 @@
             selectedItemId={selectedPlanId ?? null}
             on:deleteItem={event => deletePlanContext(event, filteredPlans)}
             on:rowClicked={({ detail }) => selectPlan(detail.data.id)}
+            on:rowDoubleClicked={showSelectedPlan}
```

Comment:
`rowDoubleClicked` should be providing the data of the row that was clicked on, so I think I would recommend utilizing the `openPlan` function instead. My concern with using `showSelectedPlan` is that it requires the selected plan, which may or may not necessarily be updated at the time `rowDoubleClicked` is fired. 

Having something like `on:rowDoubleClicked={({ detail }) => openPlan(detail.data.id)}` allows it to be pretty independent of the current state.

Reply:
@duranb yup fair.